File: 1574-maximum-product-of-two-elements-in-an-array/maximum-product-of-two-elements-in-an-array.py
class Solution:
    def maxProduct(self, nums: List[int]) -> int:
        n = len(nums)
        nums = sorted(nums, reverse = True)

        x = (nums[0] - 1)
        y = (nums[1] - 1)

        ans = (x * y)

        return ans

File: 1574-maximum-product-of-two-elements-in-an-array/maximum-product-of-two-elements-in-an-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxProduct = function(nums)
{
    nums.sort((a, b) => b - a)

    let x = nums[0] - 1
    let y = nums[1] - 1

    let ans = (x * y)

    return ans
};